01.general/18.stealth-mode/docs.en.md

```diff
@@ -68,24 +68,24 @@ When the browser addresses a page, the server assigns it an ETag, which the brow
 
 **Block WebRTC**
 
-WebRTC (Real Time Communications) is a technology that allows streaming of data directly between browsers and apps. It can let others know your real IP address even if you use a proxy or VPN.
+WebRTC (Real Time Communications) is a technology allows the streaming of data directly between browsers and apps. It can let others know your real IP address, even if you use a proxy or VPN.
 Enabling this option can disrupt the work of certain browser applications, such as messengers, chats, cinemas, or games.
 
 **Block Push API**
 
-Push API enables servers to send messages to web applications regardless of browser status. This means you may see notifications from varous websites even if your browser is hidden to tray or not launched. Enable this option to block browser Push API completely.
+The Push API enables servers to send messages to web applications regardless of browser status. This means you may see notifications from various websites even if your browser is hidden in the tray or not launched. Enable this option to block the browser's Push API completely.
 
 **Block Location API**
 
-Enabling this option will prohibit the browser from sending GPS data that could be used to find your location and modify your search results or otherwise influence your web experience.
+Enabling this option will prevent the browser from sending GPS data that could be used to find your location, modify your search results, or otherwise influence your web experience.
 
 **Block Flash**
 
-The Flash Player plugin has become increasingly vulnerable to online threats like viruses and hackers and can also have a significant negative impact on loading time for websites. When this setting is on, AdGuard blocks browsers' ability to detect components (like plugins and ActiveXObject objects) that allow Flash to display content. This effectively means that browsers are unable to support Flash.
+The Flash Player plugin has become increasingly vulnerable to online threats like viruses and hackers and can also have a significant negative impact on the loading times of websites. When this setting is on, AdGuard blocks browsers' ability to detect components (like plugins and ActiveXObject objects) that allow Flash to display content. This effectively means that browsers are unable to support Flash.
```

Comment:
```suggestion
The Flash Player plugin has become increasingly vulnerable to such online threats as viruses and hackers; it may also significantly affect website loading time. Turning this setting on makes AdGuard block browsers' ability to detect components (such as plugins and ActiveXObject objects) that allow Flash to display content. This effectively means that browsers are unable to support Flash.
```